src/typings/stripes/core.d.ts

```diff
@@ -21,6 +25,14 @@ export const IfPermission: FunctionComponent<{
   children: ReactNode | ((props: { hasPermission: boolean }) => ReactNode);
 }>;
 
+export const TitleManager: React.Component<{
```

Comment:
```suggestion
export const TitleManager: FunctionComponent<{
```

Should specify `ComponentType` or `FunctionComponent` — `Component` alone is something else unfortunately

Reply:
Ok, that fixed local errors as well. I changed to FunctionComponent.